bundle/edu.gemini.spdb.reports.collection/src/test/java/edu/gemini/dbTools/ephemeris/TestSupport.scala

```diff
@@ -33,7 +32,7 @@ trait TestSupport extends ProgramTestSupport {
 
   val genNonSiderealTarget: Gen[NonSiderealTarget] =
     Gen.oneOf(nonSids).map { case (hid, name) =>
-      NonSiderealTarget(name, ==>>.empty, Some(hid), List.empty, None, None)
```

Comment:
Here's an example as to my confusion:
I see `==>>.empty` and `IMap.empty` both used.
Why the inconsistency?